### PR TITLE
#151948277 Chore Accounts tests need fixing

### PR DESCRIPTION
**Pull request by Arthur236**
#### What does this PR do

* Fixes the tests in the accounts application

#### Description of the task to be completed

* Go through each test file and follow the instructions to fix the tests. The instructions are distributed throughout each file and the instruction lines begin with ###

#### How should this manually be tested?

* You can test this manually by issuing this command from the project root directory; 
>./manage.py test hc.accounts.tests

#### Pivotal Tracker story

* See [ Chore #151948277](https://www.pivotaltracker.com/story/show/151948277)

#### Screenshots (if appropriate)

image

#### Questions:

### CODE REVIEW

File: hc/accounts/tests/test_profile.py [hunk 1]
```diff
@@ -17,17 +27,22 @@ def test_it_sends_set_password_link(self):
         # profile.token should be set now
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
-        ### Assert that the token is set
 
-        ### Assert that the email was sent and check email content
+        #  Assert that the token is set
+        self.assertNotEquals(token, None)
```
Comment (ivantoz) [inline hc/accounts/tests/test_profile.py]: I think you can you use `assertIsNotNone`

File: hc/accounts/tests/test_profile.py [hunk 2]
```diff
@@ -107,4 +126,25 @@ def test_it_shows_badges(self):
...
+        self.assertNotEquals(api_key, None)
+
+        form = {"revoke_api_key": "1"}
+        r = self.client.post("/accounts/profile/", form)
+
+        # profile.api_key should be set none
+        self.alice.profile.refresh_from_db()
+        api_key = self.alice.profile.api_key
+        # Assert the api key is none
+        self.assertEqual(api_key, "")
```
Comment (ivantoz) [inline hc/accounts/tests/test_profile.py]: Is empty string None in python?

**Review by ivantoz — CHANGES_REQUESTED** [review]

File: hc/accounts/tests/test_profile.py (hunk 2 above)
Comment (Arthur236) [reply hc/accounts/tests/test_profile.py]: No, it's not. We just forgot to change the comment to empty string. We'll get right on it

File: hc/accounts/tests/test_profile.py (hunk 1 above)
Comment (Arthur236) [reply hc/accounts/tests/test_profile.py]: We've made the necessary changes

**Review by ivantoz — APPROVED** [review]

**Pull request merged**